integration/test/API/Brig.hs

```diff
@@ -212,3 +212,48 @@ putUserSupportedProtocols user ps = do
     baseRequest user Brig Versioned $
       joinHttpPath ["self", "supported-protocols"]
   submit "PUT" (req & addJSONObject ["supported_protocols" .= ps])
+
+getApiVersions :: HasCallStack => App Response
+getApiVersions = do
+  req <-
+    localRawBaseRequest Brig Unversioned $
+      joinHttpPath ["api-version"]
+  submit "GET" req
+
+getSwaggerPublicTOC :: HasCallStack => App Response
+getSwaggerPublicTOC = do
+  req <-
+    localRawBaseRequest Brig Unversioned $
+      joinHttpPath ["api", "swagger-ui"]
+  submit "GET" req
+
+getSwaggerInternalTOC :: HasCallStack => App Response
+getSwaggerInternalTOC = error "FUTUREWORK: this API end-point does not exist."
```

Comment:
Doesn't this simply crash? Maybe turn it into a comment.

Reply:
it's meant to crash.  nobody should call this function before it is doing anything interesting.  also, crashing isn't that bad in a test suite.  just stop calling this function, to resolve it?

Reply:
You're right, sorry, I thought this was a test. My mistake.